app/client/user/users.controller.js

```diff
@@ -96,6 +109,83 @@ angular.module('app.user')
         return vm.selection.filter(function(elem){ return elem; }).length;
       };
 
+
+      function getRole (user) {
+
+        if ( ! user.lomis_stock || ( ! user.lomis_stock.mobile || ! user.lomis_stock.dashboard) ||   ( _.isEmpty(user.lomis_stock.mobile) && _.isEmpty(user.lomis_stock.dashboard))) { return 'None'; }
+        else if (_.isEmpty(user.lomis_stock.mobile)) { return 'Dashboard'; }
+        else { return 'Mobile'; }
+      };
+
+      function getAdminLevel (user) {
+
+
+        switch ( getRole (user) ) {
+          case 'Mobile':
```

Comment:
This UMS app is supposed to be a reusable component for all couchdb related applications. In that light, consider not hard-cording any application specific parameters as you have done here and other areas.